tests/integration/create-switch-account.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { BrowserDriver, createTestSelector, setupBrowser } from './utils';
+import { WalletPage } from './page-objects/wallet.page';
+import { ScreenPaths } from '@common/types';
+import { SettingsSelectors } from './settings.selectors';
+
+jest.setTimeout(40_000);
+jest.retryTimes(process.env.CI ? 2 : 0);
+describe.only(`Create and switch account`, () => {
+  let browser: BrowserDriver;
+  let wallet: WalletPage;
+
+  beforeEach(async () => {
+    browser = await setupBrowser();
+    wallet = await WalletPage.init(browser, ScreenPaths.INSTALLED);
+  }, 10000);
+
+  afterEach(async () => {
+    try {
+      await browser.context.close();
+    } catch (error) {}
+  });
+
+  it('should be able to create a new account then switch', async () => {
+    await wallet.signUp();
+    await wallet.clickSettingsButton();
+    await wallet.page.click(createTestSelector('settings-create-an-account'));
+    await wallet.page.waitForSelector(createTestSelector('create-account-done-button'))
+    await wallet.page.click(createTestSelector('create-account-done-button'));
+    const displayName = await wallet.page.textContent(
+      createTestSelector('home-current-display-name')
+    );
+    expect(displayName).toEqual('Account 2');
+    await wallet.clickSettingsButton();
+    await wallet.page.click(createTestSelector(SettingsSelectors.SwitchAccount));
+    await wallet.page.click(createTestSelector('switch-account-item-0'));
+    await wallet.page.waitForSelector(createTestSelector('account-checked-1'), {state: 'hidden'})
+    const displayName2 = await wallet.page.textContent(
+      createTestSelector('home-current-display-name')
+    );
+    expect(displayName2).toEqual('Account 1');
+  });
+
+  const COUNT = 3;
+  it(`should be able to create ${COUNT} new accounts then switch between them`, async () => {
```

Comment:
Unless we are mocking the network requests, every time this seed phrase generates a new account, it will be persisted across sessions, meaning indexes 0-2 will only ever exist on the first run

Reply:
I did not know that, I will check it out.

Reply:
@aulneau Do you mean that it will create those 3 accounts even if we sign in with a **different** seed phrase ?

Reply:
this might help:
- user signs in with new seed phrase
- user has 1 "Account"
- user hits "create account" 3 times
- user now has 4 accounts
- user signs out completely
- user signs in with previous seed phrase
- user hits "create account"
- user now has 5 accounts

if a user were to use a "fresh" seed phrase each time, there would be no persistence of accounts

Reply:
So there is no issue in this test because we create a new seed phrase just for this. 
See [signUp()](https://github.com/blockstack/stacks-wallet-web/pull/1369/files#diff-b9f26664d0f0636a3e738d71d1702759930b947553e91be302081d96462945d7R45)
The accounts won't be persisted across tests. Let me know if I am wrong somewhere.

Reply:
Yep, you're right, perfect!